components/PlayerIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react'
+import { Image, Text, View } from 'react-native'
+
+import crown from '../assets/crown.png'
+
+interface Props {
+  name?: String;
+  index: any;
+}
+
+export const PLAYER_IMAGES = [
+  { uri: require('../assets/player-1.png'), color: 'bg-[#f19f3f]' },
+  { uri: require('../assets/player-2.png'), color: 'bg-[#eebebb]' },
+  { uri: require('../assets/player-3.png'), color: 'bg-[#fbe9e7]' },
+  { uri: require('../assets/player-4.png'), color: 'bg-[#F7D2A2]' }
+]
+
+const PlayerIcon: FunctionComponent<Props> = (props) => {
+  const { name, index } = props;
+  let imgSource = PLAYER_IMAGES[index].uri;
+  const even = index % 2 == 0;
+  let gridPosition = even ? 'py-2' : 'mt-8 mb-4';
+  let position = even ? 'translate-x-4' : '-translate-x-4';
+  let roundedness = even ? 'rounded-l-2xl' : 'rounded-r-2xl';
+  let translate = even ? 'translate-x-4' : '-translate-x-2';
+
+  return (
+    <View className={`${gridPosition} h-40 items-center justify-center relative w-1/2`}>
+      {index === 0 &&
```

Comment:
we have another mean of checking if the user is the `host` (see the `isHost` variable in the `Matchmaking` component)